packages/ui/src/helpers/authenticator/__tests__/getRoute.test.ts

```diff
@@ -0,0 +1,95 @@
+import {
+  AuthActorState,
+  AuthMachineState,
+} from '../../../machines/authenticator/types';
+
+import { getRoute } from '../getRoute';
+
+describe('getRoute', () => {
+  let mockState: AuthMachineState;
+  let mockActorState: AuthActorState;
+
+  beforeEach(() => {
+    mockState = {
+      value: 'unknown',
+      matches: function (this: AuthMachineState, targetState: string) {
+        return this.value === targetState;
+      },
+    } as AuthMachineState;
+
+    mockActorState = {
+      value: 'unknown',
+      matches: function (this: AuthActorState, targetState: string) {
+        return this.value === targetState;
+      },
+    } as AuthActorState;
```

Comment:
Since `state` and `actorState` are xstate utils, prefer creating spies over mocks that could potentially drift from their base implementation

Reply:
Discussed offline and decided to create state/actor state within each test with both values set to what they would actually be if running.